lib/bulk_insert/worker.rb

```diff
@@ -113,7 +113,16 @@ def compose_insert_query
     end
 
     def insert_sql_statement
-      insert_ignore = 'IGNORE' if (adapter_name == 'MySQL') && ignore
+      insert_ignore = if ignore
+        if adapter_name == "MySQL"
+          'IGNORE'
+        elsif adapter_name.match(/sqlite.*/)
```

Comment:
Should this be a case-insensitive match? When I connect ActiveRecord to a SQLite database, it tells me that the adapter name is `SQLite`. Perhaps the regex should be `/sqlite/i`?